app/component/summary/summary-row.cjsx

```diff
@@ -48,7 +51,7 @@ class SummaryRow extends React.Component
           {geoUtils.displayDistance(data.walkDistance)}
         </div>
       </div>
-      <div className="itinerary-start-time">
+      <div className={cx "itinerary-start-time", if realTimeAvailable then "realtime-available"}>
```

Comment:
You can just use `{cx "itinerary-start-time", "realtime-available": realTimeAvailable}` to simplify a bit